app/src/main/res/values/strings.xml

```diff
@@ -399,7 +399,7 @@
     <!-- Summary of tracking protection preference if tracking protection is set to off -->
     <string name="tracking_protection_off">Off</string>
 
-
     <!-- DEPRECATED -->
     <string name="tabs_header_title">DEPRECATED</string>
+    <string name="session_save">DEPRECATED</string>
```

Comment:
Is this going to cause a problem because we exported strings already? How do we communicate with translators that a translation isn't needed anymore?

Reply:
We're still working that out. Unfortunately I can't just take them our right now because we get a lot of linting errors. :(